templates/webhook.bin.erb

```diff
@@ -108,7 +114,13 @@ class Server < Sinatra::Base
     data = JSON.parse(decoded, :quirks_mode => true)
 
     # github sends a 'ref', stash sends an array in 'refChanges'
-    branch = ( data['ref'] || data['refChanges'][0]['refId'] ).sub('refs/heads/', '')
+    if $config['git_provider'] == 'github'
+      branch = ( data['ref'] || data['refChanges'][0]['refId'] ).sub('refs/heads/', '')
```

Comment:
Yeah I gathered that when I looked at my Gitlab environment, I needed a variable name to standardize on, so I chose GitHub as in "GitHub compliant," figured githublabstash would be too messy :) I am certainly open to suggestions of change.